Sofa/Component/Collision/Detection/Intersection/src/sofa/component/collision/detection/intersection/MinProximityIntersection.cpp

```diff
@@ -45,12 +45,18 @@ int MinProximityIntersectionClass = core::RegisterObject("A set of methods to co
 
 MinProximityIntersection::MinProximityIntersection()
     : BaseProximityIntersection()
-    , useSphereTriangle(initData(&useSphereTriangle, true, "useSphereTriangle","activate Sphere-Triangle intersection tests"))
-    , usePointPoint(initData(&usePointPoint, true, "usePointPoint","activate Point-Point intersection tests"))
-    , useSurfaceNormals(initData(&useSurfaceNormals, false, "useSurfaceNormals", "Compute the norms of the Detection Outputs by considering the normals of the surfaces involved."))
-    , useLinePoint(initData(&useLinePoint, true, "useLinePoint", "activate Line-Point intersection tests"))
-    , useLineLine(initData(&useLineLine, true, "useLineLine", "activate Line-Line  intersection tests"))
+    , d_useSphereTriangle(initData(&d_useSphereTriangle, true, "d_useSphereTriangle", "activate Sphere-Triangle intersection tests"))
+    , d_usePointPoint(initData(&d_usePointPoint, true, "usePointPoint", "activate Point-Point intersection tests"))
+    , d_useSurfaceNormals(initData(&d_useSurfaceNormals, false, "useSurfaceNormals", "Compute the norms of the Detection Outputs by considering the normals of the surfaces involved."))
+    , d_useLinePoint(initData(&d_useLinePoint, true, "d_useLinePoint", "activate Line-Point intersection tests"))
```

Comment:
```suggestion
    , d_useLinePoint(initData(&d_useLinePoint, true, "useLinePoint", "activate Line-Point intersection tests"))
```

Can you look over all your PRs that there are no such mistakes?